Runtime/Scripts/MobiledgeXIntegrationConfig.cs

```diff
@@ -88,6 +88,10 @@ public static async Task<LocationFromIPAddress> GetLocationFromIP()
         HttpResponseMessage response = await httpClient.GetAsync("https://freegeoip.app/json/").ConfigureAwait(false);
         string responseBodyStr = response.Content.ReadAsStringAsync().Result;
         LocationFromIPAddress location = Messaging<LocationFromIPAddress>.Deserialize(responseBodyStr);
+        if (location.latitude == 0 && location.longitude == 0)
+        {
+          throw new Exception("Location API returned {0,0}, overriding location");
```

Comment:
It says overriding, but what is the value used for the override?

Reply:
Thanks.
Fixed.